src/content/github.js

```diff
@@ -1,7 +1,8 @@
-import { findWithClass, isElementVisible } from "./utils";
+import { findWithClass, isElementVisible } from "./dom-utils";
```

Comment:
I'd prefer naming the file `domUtils.js` to be consistent with existing files like `stackOverflow.js`.